.travis.yml

```diff
@@ -14,11 +14,11 @@ before_install:
   - echo "https://$GH_TOKEN:@github.com" > .git/credentials
   - gem install asciidoctor
 install:
-- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=true
+- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=false
```

Comment:
I'm not sure I understand how changing this to `false` is going to reduce log noise, since redirecting test output to a file vs. stdout would seem to be the correct settings.